gitlab/gitlab_exporter.py

```diff
@@ -136,11 +139,15 @@ def get_all_entities_from_gitlab():
     response = requests.get(
         api_url,
         headers={"PRIVATE-TOKEN": GITLAB_API_TOKEN},
+        # By default, this request returns 20 results at a time because the API results are paginated.
+        # Archived projects are included by default so we need to exclude them.
+        params={'include_subgroups': True, 'archived': False, 'per_page': 100}
```

Comment:
How does this iterate over the additional pages if there are more than 100 results?